server/src/main/java/com/distkv/server/metaserver/client/DmetaClient.java

```diff
@@ -48,9 +50,8 @@ public HeartbeatResponse heartbeat(NodeInfo nodeInfo) {
           .invokeSync(leader.getEndpoint().toString(), request, HEARTBEAT_TIMEOUT);
       if (!response.isSuccess()) {
         if (response.getRedirect().length() > 0) {
-          PeerId peerId = new PeerId();
-          peerId.parse(response.getRedirect());
-          RouteTable.getInstance().updateLeader(RAFT_GROUP_ID, peerId);
+          refreshLeader();
```

Comment:
？

Reply:
I think the frequency of using this kind of operation is very low, and the difference between using a time-consuming method is not big.